src/app/tests/suites/certification/Test_TC_SC_4_5.yaml

```diff
@@ -26,7 +26,7 @@ tests:
       verification:
           "1. On the raspi controller, publish matter service, using below
           command $avahi-publish-service 87E1B004E235A130-8FC7772401CD0696
-          _matter._tcp 22222 CRI=3000 CRA=4000 T=0"
+          _matter._tcp 22222 SII=3000 SAI=4000 T=0"
```

Comment:
Having a SAI > SII seems... odd.  It sort of made sense maybe when these were retry intervals, but now that these are supposed to be the sleep periods, this makes no sense.